streaming/index.js

```diff
@@ -1210,8 +1206,8 @@ const startServer = async () => {
 
       break;
     case 'hashtag':
-      if (!params.tag || params.tag.length === 0) {
```

Comment:
I realised that these two clauses were functionally equivalent, hence dropping the length check.